Lesson23/Kadir/filter.js

```diff
@@ -353,6 +362,9 @@ const studentsExercise9 = [
 	},
 ];
 
+const studentsExercise9Result = studentsExercise9.filter((student) => !student.grade.includes('D'));
```

Comment:
Good but this would only work for this particular set of data. In real word you need to think about all other potential cases like C−, D+, D, D−, and F.